Config/AltBuildSystems/testTSG.py

```diff
@@ -146,13 +146,17 @@ def testBasicIO(self):
         print("                Available GPUs:")
         if (grid.getNumGPUs() > 0):
             for iGPU in range(grid.getNumGPUs()):
-                sName = grid.getGPUName(iGPU)
+                sName = ""
+                if ("GNU" == "MSVC"):
```

Comment:
????

Reply:
The file is automatically generated from `if (@CMAKE_CXX_COMPILER_ID@ == "MSVC")`

Had to disable one of the tests in Visual C++, passing null terminated strings from C++ to Python was crashing, but it could have been something to do with CUDA. This is only the test script, not the actual Python module. I wouldn't put that in the module.